### DISCUSSION TITLE
jersey-server dependency

### DISCUSSION THREAD

**Discussion by georgweiss** [ControlSystemStudio/phoebus#2093] [Q&A]
The phoebus-target pom lists dependency to jersey-server. Is this really needed?

**Comment by shroffk** [comment]
Unless the server is pulled in as a third party I don't think we need it.
Since the services use springboot, which is a better replacement for the jersey-server I think we can look at trying to remove it

**Reply by georgweiss** [reply]
I could not see any signs of this being pulled in by some other dependency.
